app/controllers/admin/events/list.js

```diff
@@ -128,10 +128,16 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       this.refreshModel.bind(this)();
     } catch (e) {
       console.error('Error while deleting event', e);
-      this.notify.error(this.l10n.t('An unexpected error has occurred.'),
-        {
-          id: 'event_delete_error'
+      if ('errors' in e && e.errors.length > 0) {
```

Comment:
Can be reduced to `e.errors?[0]?.detail`

Reply:
Done.Thanks for the review.